omero/sysadmins/server-upgrade.txt

```diff
@@ -59,6 +59,13 @@ your previous version of OMERO. We recommend using a future-proof symlink
 if possible, so that these stanzas do not need updating with each OMERO
 server upgrade.
 
+.. note:: Since OMERO 5.2, the OMERO.web framework no longer bundles
+    a copy of the Django package, instead manual installation of
+    the Django dependency is required. It is highly recommended to use
+    `Django 1.8`_ (LTS) which requires Python 2.7. For more information
+    see :ref:`python-requirements` on the
+    :doc:`/sysadmins/version-requirements` page.
```

Comment:
I'm wondering if we need separate links both to a section and to the page the section's on? Though if you think it adds value I'm fine with it.